hc/api/tests/test_create_check.py

```diff
@@ -49,23 +49,23 @@ def test_it_accepts_api_key_in_header(self):
         payload = json.dumps({"name": "Foo"})
 
         ### Make the post request and get the response
-        r = self.post({
+        response = self.post({
             "api_key": "abc",
             "name": "Foo"
             })
-        self.assertEqual(r.status_code, 201)
+        self.assertEqual(response.status_code, 201)
 
     def test_it_handles_missing_request_body(self):
         ### Make the post request with a missing body and get the response
-        r = self.post({"api_key": ""}) 
-        self.assertEqual(r.status_code, 400)
-        self.assertEqual(r.json()["error"], "wrong api_key")
+        response = self.post({"api_key": ""}) 
+        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.json()["error"], "wrong api_key")
 
     def test_it_handles_invalid_json(self):
         ### Make the post request with invalid json data type
-        r = {'status_code': 400, 'error': "could not parse request body"} ### This is just a placeholder variable
-        self.assertEqual(r['status_code'], 400)
-        self.assertEqual(r["error"], "could not parse request body")
+        response = {'status_code': 400, 'error': "could not parse request body"} ### This is just a placeholder variable
+        self.assertEqual(response['status_code'], 400)
+        self.assertEqual(response["error"], "could not parse request body")
 
     def test_it_rejects_wrong_api_key(self):
         responce = self.post({"api_key": "wrong"},
```

Comment:
This is a spelling mistake. Please correct it to `response`.

Reply:
Thanks had missed that